new/Bear/server.js

```diff
@@ -0,0 +1,79 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+
+const Bear = require('./Bears/BearModel.js');
+
+const server = express();
+
+server.listen(5000, function() {

```

Comment:
Ideally you want this to be the last part of your file. You want to initialize everything your server needs first, like the body parser, the routes, etc. After that then you can tell your server to listen.